pandas/core/series.py

```diff
@@ -3238,6 +3238,8 @@ def _reduce(self, op, name, axis=0, skipna=True, numeric_only=None,
 
         """
         delegate = self._values
+        if axis is None:
+            axis = self._stat_axis_number
```

Comment:
Why is this still needed?

Reply:
Can you have a look at this one?

Reply:
Sorry, missed this earlier. We still call `self._get_axis_number(axis)` to validate that the user passed `axis` is correct. I'll modify things to be clearer.